tests/tests.go

```diff
@@ -473,8 +473,8 @@ func setWebhookOptMode(webhookName, optMode string) error {
 func addFinalizer(virtualMachine *kubevirtv1.VirtualMachine, finalizerName string) error {
 	err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 		var err error
-		virtualMachine, err = testClient.VirtClient.VirtualMachine(virtualMachine.Namespace).Get(context.TODO(),
-			virtualMachine.Name, metav1.GetOptions{})
+		err = testClient.CRClient.Get(context.TODO(),
```

Comment:
context.TODO is deprecated, please use context.Background. Here and in other occurrences.

Dont we have linter rule for that? :thinking: